.travis.yml

```diff
@@ -6,3 +6,6 @@ jdk:
 #workaround for https://github.com/travis-ci/travis-ci/issues/5227
 addons:
   hostname: dropwizard-jobs-ci
+
+before_install:
+  - chmod +x mvnw

```

Comment:
why is this needed? git should be able to store permissions if you set it or am I wrong?